simapp/app.go

```diff
@@ -317,6 +318,13 @@ func NewSimApp(
 	// If evidence needs to be handled for the app, set routes in router here and seal
 	app.EvidenceKeeper = *evidenceKeeper
 
+	/****  Module Options ****/
+	var skipGenesisInvariants = false
+	opt := appOpts.Get(crisis.FlagSkipGenesisInvariants)
+	if opt, ok := opt.(bool); ok {
+		skipGenesisInvariants = opt
+	}
```

Comment:
I was thinking if it's better to assign options here or passing `appOpts` to `NewAppModule`. Thoughts?
 
Let me know and I will update `gaia` accordingly.

Reply:
If we require `appOpts` only for one module, may be passing `appOpts` to `NewAppModule` is better.

Reply:
Sure

Reply:
Although there's likely a better way. I'd like to see much leaner App constructors. But for post-stargate...

Reply:
After having a second look, I prefer to be more explicit with module constructor rather than parsing `appOps` which is very general. Let's see how this will evolve in the future. I added a note to the code.